controllers/api/tutorRoutes.js

```diff
@@ -1 +1,57 @@
-// POST /api/tutor - Create a new tutor
\ No newline at end of file
+const router = require("express").Router();
+const { withAuthJson } = require("../../utils/auth");
+const { Tutor } = require("../../models");
+
+
+
+router.post("/", withAuthJson, async (req, res) => {
```

Comment:
This was crashing on my when I was submitting a bad email or a duplicate email, so a try-catch will help keep it alive for error cases:

```router.post("/", withAuthJson, async (req, res) => {
    try {
        const tutorData = await Tutor.create(req.body);
        const tutor = tutorData.toJSON();
        res.json(tutor);   
    }
    catch (err) {
        res.status(400).json(err);
    }
  });```

Reply:
I put a change on a branch here, if you want to cherry-pick it, or copy it: a7650cc4625f3b3ab361e8a7dca28a188da2e6b5

Reply:
ok sweet